docs/UserGuide.md

```diff
@@ -31,104 +31,238 @@
 
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
+> * Items in `[SQUARE_BRACKETS]` are optional.
 > * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+> * The order of parameters is not fixed.
 
 #### Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task or event: `add`
 
-> Persons can have any number of tags (including 0)
+Adds a task to Lifekeeper<br>
+Format: `add TASK_NAME [d/DEADLINE] [p/PRIORITY_LEVEL] [r/REMINDER] [t/TAG]...` 
+
+> Tasks can have any number of tags (including 0)
+
+`DEADLINE` accepts `Date Time` format input and variable inputs namely:
+* `tomorrow [TIME]`
+* `today [TIME]`
+
+`PRIORITY_LEVEL` has to be an integer equal or larger than 1, with `1` being the top priority.
+
+`REMINDER` accepts `Date Time` format input and variable inputs namely:
+* `tomorrow [TIME]`
+* `today [TIME]`
+* `[TIME] before` sets reminder at the specified time before the `DEADLINE`. e.g. `0015 before` for a reminder 15 minutes before the `DEADLINE`.
+
+Examples: 
+* `add Grocery Shopping`
+* `add Assignment 1 d/Tomorrow p/1 r/Today 2000`
+* `add Assignment 2 d/Tomorrow p/1 r/0400 before`
+* `add CS2103 T7A1 d/6 Oct 2016 p/2 r/5 Oct 2016 1800 t/teamC2`
+
+Adds an event to Lifekeeper<br>
+Format: `add EVENT_NAME s/START_TIME e/END_TIME [t/TAG]...` 
+
+> Events can have any number of tags (including 0)
+
+`START_TIME` and `END_TIME` accepts `Date Time` format input and variable inputs namely:
+* `tomorrow [TIME]`
+* `today [TIME]`
+* `TIME` format assumes `today`
+
+Examples: 
+* `add Lunch s/1200 e/1300`
+* `add Executive Meeting s/tomorrow 0900 e/tomorrow 1200`
+* `add Concert s/tomorrow 1800 e/tomorrow 2000 t/Leisure`
+
+#### Adding a category: `addcat`
```

Comment:
once you add a category, how do you assign to the task? 
what if you want to assign some category to the task that is not added yet? 
can you think of eliminating this totally, and use an option parameter like tag in the add command? 
on second thoughts, why not use tag itself to mark categories, in which case, this command seems like unnecessary duplicate work.
do think on these lines.